src/main/java/TaskList.java

```diff
@@ -0,0 +1,42 @@
+public class TaskList {
+    private Task[] taskList = new Task[100];
```

Comment:
array specifiers are correctly attached to the type! (Task[] instead of Task taskList[]) 
good job!